Dockerfile

```diff
@@ -1,11 +1,21 @@
-FROM node:latest
-RUN apt-get update && apt install git -y
+# Fetch and build the node modules
+FROM node:13-alpine as BUILD_IMAGE
 
-RUN git clone https://github.com/Danielv123/screepsPrometheus.git && cd screepsPrometheus && npm install
+RUN apk update && apk add python make g++ && rm -rf /var/cache/apk/*
+
+WORKDIR /screepsPrometheus
+COPY package.json package-lock.json ./
+RUN npm install
+
+# Setup the actual run environment
+FROM node:13-alpine
+
+WORKDIR /screepsPrometheus
+COPY --from=BUILD_IMAGE /screepsPrometheus/node_modules ./node_modules
+COPY README.md LICENSE src/ ./
```

Comment:
I am curious, why don't you just delete those files? They don't need to be in the image, right?

Reply:
I left them in the image so they would be there when/if people go snooping. I guess you're right that they are not needed.